examples/kitchen-sink-vite/app/react-worker/host/remote-renderer.tsx

```diff
@@ -0,0 +1,35 @@
+import React, {useMemo, useEffect} from 'react';
+import Worker from '../remote/worker?worker';
+
+import {
+  createController,
+  createRemoteReceiver,
+  RemoteRenderer as OriginalRemoteRenderer,
+} from '@remote-ui/react/host';
+
+import {createEndpoint, fromWebWorker} from '@remote-ui/rpc';
+
+import {Button} from './components';
+import {Endpoint} from '../types';
+
+const remoteEndpoint = createEndpoint<Endpoint>(fromWebWorker(new Worker()));
+
+export function RemoteRenderer({
```

Comment:
I think we should find a different name for this to avoid conflicting with the library's component name.